pkg/member/member.go

```diff
@@ -302,7 +303,7 @@ func (m *Member) getMemberLeaderPriorityPath(id uint64) string {
 
 // GetDCLocationPathPrefix returns the dc-location path prefix of the cluster.
 func (m *Member) GetDCLocationPathPrefix() string {
-	return path.Join(m.rootPath, dcLocationConfigEtcdPrefix)
+	return path.Join(m.rootPath, endpoint.MicroserviceKey, endpoint.TSOServiceKey, "default", dcLocationConfigEtcdPrefix)
```

Comment:
I think `endpoint.MicroserviceKey` + `service_key` could be extracted by a common function to be used later maybe.

Reply:
sounds good